src/customers-container/customers-store.js

```diff
@@ -73,8 +73,11 @@ class CustomersStore {
         customerInfo.url,
         ''
       );
-      await this.makeHttpRequest(UPDATE_CUSTOMER_QUERY);
-      this.fetchCustomers();
+      const response = await this.makeHttpRequest(UPDATE_CUSTOMER_QUERY);
+      if (response.updateCustomer) {
+        alert('update customer successfully');
```

Comment:
no alert

Reply:
proper notification is yet to be implemented

Reply:
comment out, put TODO: add notification, same w error handling

Reply:
done